wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel.swift

```diff
@@ -250,6 +260,17 @@ extension ConversationListViewController.ViewModel {
     }
 }
 
+// MARK: - Observing certificate changes
+
+extension ConversationListViewController.ViewModel {
+
+    @objc
+    private func updateE2EICertificateStatus() {
+        updateE2EICertifiedStatus()
+    }
+
+}
```

Comment:
Personal preference: isn't it easier to avoid these extensions (extensions were designed for different purpose in my opinion) and just use the `// MARK` comments to structure code?

```suggestion
struct ConversationListViewController.ViewModel {
    
    // ... code ...
    
    // MARK: - Observing certificate changes

    @objc
    private func updateE2EICertificateStatus() {
        updateE2EICertifiedStatus()
    }
}
```